electricpy/__init__.py

```diff
@@ -6573,9 +6573,277 @@ def wireresistance(length=None,diameter=None,rho=16.8*10**-9,R=None):
         A = rho*length/R
         return _np.sqrt(4*A/pi)
         
+def ic_555(mode,R=None,C=None,f=None,t_high=None,t_low=None):
+
+    if mode == 'ASTABLE':
+
+        if R!=None and C!=None:
+
+            try:
```

Comment:
Instead of this try/except block, since we're merely asserting, let's look at making it a conditional instead, something like:

```python
if len(R) != 2:
    raise ValueError(f"ic_555 in {mode} will have only 2 resitances to be fixed but {len(R)} were \
                                given")
```

Reply:
done!!! 😀😀